test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -612,4 +619,39 @@ func TestSRIOVNetwork(t *testing.T) {
 	if err := testData.assertPodNetworkStatus(t, clientset, pods, ns, true); err != nil {
 		t.Fatalf("Error when checking the Pod annotation: %v", err)
 	}
+	testData.assertVFName(t, e2eTestData, vfName, pod1, node1, ns)
+}
+
+func (data *testData) assertVFName(t *testing.T, e2eTestData *antreae2e.TestData, vfName, podName, nodeName, ns string) {
+	//  Delete a Pod and check the VF device is recovered with the original interface name.
+	err := e2eTestData.DeletePodAndWait(10*time.Second, podName, ns)
+	require.NoError(t, err, "Unable to delete the Pod %s/%s in time", ns, podName)
+	recoveredVFName := GetVFInterfaceName(t, e2eTestData, nodeName)
+	logs.Infof("The recovered VF interface name is %s on Node %s", recoveredVFName, nodeName)
+	assert.Equal(t, vfName, recoveredVFName, "VF name is not recovered correctly on Node %s, the expected VF name is %s, but got %s", nodeName, vfName, recoveredVFName)
+}
+
+func GetVFInterfaceName(t *testing.T, e2eTestData *antreae2e.TestData, nodeName string) string {
+	cmd := []string{"ip", "-d", "link", "show"}
+	stdOut, _, err := e2eTestData.RunCommandFromAntreaPodOnNode(nodeName, cmd)
+	if err != nil {
+		require.NoError(t, err, fmt.Sprintf("Error when checking the VF interface name on the Node %s", nodeName))
+	}
+
+	lines := bytes.Split([]byte(stdOut), []byte("\n"))
```

Comment:
should you just convert `stdOut` to `string` right away and then use the `strings` functions, such as https://pkg.go.dev/strings#Lines?

Reply:
Done, comment addressed in the top commit.